File: TP/TP2/Code/4C.cpp
/* Afficher N en enversant l'ordre des chiffres. N est saisi par USER */
#include <iostream>

int main()
{
  int n;
  int temp;
  
  std::cout << "Donne moi une valeur : " << std::flush;
  std::cin >> n;
  
  while (n > 0) {
    temp = n % 10;
    n /= 10;
    std::cout << temp << std::flush;
  }
  
  std::cout << std::endl;
  return 0;
}

File: TP/TP2/Code/Python/4C.py
#!/usr/bin/python
# Afficher N en enversant l'ordre des chiffres. N est saisi par USER
print('Donne moi une valeur : ', end='')
foo = int(input())

while foo > 0:
    temp = foo % 10
    foo //= 10  # I ❤ Python
    print(temp, end='')
